### DISCUSSION TITLE
wrong reduced_chi_square

### DISCUSSION THREAD

**Discussion by sfzastro** [mboquien/cigale#23] [General]
Hi,

I use 31 filters in my fitting, and not all objects have so many valid fluxes. 
However, in the fitting results, the best.chi_square / best.reduced_chi_square is 30 for all my objects.

`CIGALE version: 2022.1 - Python version: 3.8.12 - Platform: linux-x86_64 `

**Comment by Guang91** [comment]
Have you followed the figure for invalid fluxes/errors? (the figure is from the CIGALE manual)
![image](https://user-images.githubusercontent.com/7922328/187299255-ec9b6ec2-270a-4bfa-bab7-948b278b7dd8.png)

**Reply by sfzastro** [reply]
I set the invalid fluxes to -99, which no longer works.
Thanks.

**Reply by mboquien** [reply]
The easiest way is to put flux and uncertainties to NaN. That way, there is no ambiguity.
